synapse/federation/federation_server.py

```diff
@@ -678,6 +679,7 @@ async def on_send_join_request(
         room_id: str,
         caller_supports_partial_state: bool = False,
     ) -> Dict[str, Any]:
+        set_tag("partial_state", caller_supports_partial_state)
```

Comment:
Note that tags are globally searchable. There is an idea (more honoured in the breach than the observance) that tags should use symbolic constants defined  in the `SynapseTags` class, where we can also put a comment giving a good definition of what the tag actually means.

(In this case: does it mean that the request is for a partial state response, or that the response is partial state, or that the room is currently partial-stated, or?)

An alternative is to use `log_kv` to log a key/value pair. They aren't searchable and therefore there's less need to clearly define their semantics.

Reply:
In this situation I want something searchable. How does 083a659b4a look?